gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlinVersion=1.2.20
-junitPlatformVersion=1.0.1
+junitPlatformVersion=1.1.0
```

Comment:
WDYT about moving all versions to a single place?

Reply:
PR welcome imo, not critical fro this one in particular

Create issue? :)

Reply:
#375